index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt("Ingrese su nombre");
+alert('Hola ${nombreUsuario}!');
```

Comment:
acá se podría hacer una corrección, por ejemplo: En este caso no seria bueno utilizar interpolación de string.

```suggestion
alert('Hola ' + nombreUsuario + '!');
```